scripts/provision.sh

```diff
@@ -200,7 +201,22 @@ start_datastore() {
 }
 
 start_minio() {
+  set +e
+  is_minio_migration_needed
+  migration_needed=$?
+  set -e
+  
+  if [ "$migration_needed" -eq 1 ]; then
+    echo MinIO migration required
+    bash ./minio-update.sh download
+    sudo sh -c "find /hab/svc/builder-minio/data/ -maxdepth 1 -mindepth 1 -type d | xargs rm -rf"
```

Comment:
I would move this to some other location and output a message that their old minio data was backed up to that location and if their upgrade is successful then it can be deleted.